tests/e2e/core-tests/package.json

```diff
@@ -11,7 +11,8 @@
   "main": "index.js",
   "dependencies": {
     "@jest/globals": "^26.4.2",
-    "config": "3.3.3"
+    "config": "3.3.3",
+    "faker": "^5.5.2"
```

Comment:
Is faker used in this test? If not can we remove this addition

Reply:
Oh right! Yeah, I just realized this test doesn't require faker anymore. Thanks for spotting this @tammullen 

I removed it from the `/core-tests/package.json` file now

Reply:
Thanks!